politeiad/backendv2/tstorebe/plugins/comments/comments.go

```diff
@@ -102,12 +102,36 @@ func (p *commentsPlugin) Hook(h plugins.HookT, payload string) error {
 func (p *commentsPlugin) Fsck(tokens [][]byte) error {
 	log.Tracef("comments Fsck")
 
-	// Verify the cohereny of the record index for each token. This
-	// includes verifying that:
-	// - All comment adds are present in the record index.
-	// - All comment dels are present in the record index and the
-	//   corresponding comment adds have been deleted from the db.
-	// - All comment votes are present in the record index.
+	log.Infof("Starting comments fsck for %v records", len(tokens))
+
+	// Navigate the provided record tokens and verify their record index
+	// comments cache integrity. This will only rebuild the cache if any
+	// inconsistency is found.
+
+	counter := 0 // number of rebuilded record indexes
```

Comment:
Don't initialize variables to their zero value with short declarations

`var counter int`